__tests__/cron/cleanGiftedPlus.ts

```diff
@@ -0,0 +1,70 @@
+import cron from '../../src/cron/cleanGiftedPlus';
+import { expectSuccessfulCron, saveFixtures } from '../helpers';
+import { User } from '../../src/entity';
+import { plusUsersFixture, usersFixture } from '../fixture';
+import { DataSource, JsonContains } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { addDays, subDays } from 'date-fns';
+import { SubscriptionCycles } from '../../src/paddle';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+beforeEach(async () => {
+  await saveFixtures(con, User, [...usersFixture, ...plusUsersFixture]);
+});
+
+it('should remove expired gifted plus subscription', async () => {
+  const yesterdayDate = subDays(new Date(), 1);
+  const tomorrowDate = addDays(new Date(), 1);
+
+  // 5 total users:
+  // - 1 is gifted and expired;
+  // - 2 is gifted and NOT expired;
+  // - 3 & 4 are not plus.
+  // - 5 is classic yearly plus;
+  await con.getRepository(User).update(
+    { id: '1' },
```

Comment:
can you insert users with prefix eg. `1-clgp` for this test spec, this is so that they don't overlap with other tests and become flaky? 

Check here for example https://github.com/dailydotdev/daily-api/commit/2666bca4c84875e8e12d25c3da3c30ba51b750f0#diff-6dda666a738903e7b6ff809e0a3fb7dff8a41b86a053d1c1d278f15df92e274dR31-R42

Reply:
Updated ✔️